site/content/installation/installing-ngf/manifests.md

```diff
@@ -46,80 +46,99 @@ kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric
 
 {{<note>}}By default, NGINX Gateway Fabric is installed in the **nginx-gateway** namespace. You can deploy in another namespace by modifying the manifest files.{{</note>}}
 
-{{<note>}}If you are deploying NGINX Gateway Fabric on OpenShift, we've supplied a baseline [SecurityContextConstraints](https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.3.0/deploy/manifests/scc.yaml) manifest that you can download, modify if needed, and apply. You will also need to make sure the SecurityContextConstraints resource is added to the ClusterRole RBAC:
-
-```yaml
-. . .
-- apiGroups:
-  - security.openshift.io
-  resources:
-  - securitycontextconstraints
-  resourceNames:
-  -  nginx-gateway-scc
-  verbs:
-  - use
- ```
-
-Alternatively, use [helm]({{< relref "installation/installing-ngf/helm.md" >}}), which will automatically configure the mentioned resources on OpenShift.
-{{</note>}}
+{{<tabs name="install-manifests">}}
 
-#### Stable release
+{{%tab name="Default"%}}
 
-##### For NGINX
+Deploys NGINX Gateway Fabric with NGINX OSS.
 
 ```shell
-kubectl apply -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.3.0/nginx-gateway.yaml
+kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.3.0/default/deploy.yaml
```

Comment:
So we will be publishing these as part of the release? Do they need to be a part of goreleaser?

Reply:
They will just be in the repo, not part of the release. Do you think we still need to upload them to the release too?

Reply:
Oh, I see. No, that's fine. Will this link work, though? Looks like it is missing the `deploy` directory. Same with the other links.